xup/xup.c

```diff
@@ -212,7 +212,7 @@ lib_mod_connect(struct mod *mod)
         if (trans_connect(mod->trans, mod->ip, con_port, 3000) == 0)
         {
             LLOGLN(0, ("lib_mod_connect: connected to Xserver "
-                   "(Xorg or X11rdp) sck %ld", mod->trans->sck));
+                   "(Xorg or X11rdp) sck %ld", (long) (mod->trans->sck)));
```

Comment:
I'm surprised why `sck` is not defined as `int` in `common/trans.h`. If it's an opaque pointer, it should be printed as a pointer. If it needs to be a 64-bit integer, let's print it as `unsigned long long`.